src/index.html

```diff
@@ -5,9 +5,319 @@
+              class="person__photo"
+            >
+            <h3 class="person__name">
+              Katherine Adams
+            </h3>
+            <p class="person__position">
+              Senior Vice President
+            </p>
+            <p class="person__text">
+              My soul is illuminated by an unearthly joy, as these beautiful spring morning, which I enjoy with all my heart. I’m all alone and blissfully happy in the local edge.
+            </p>
+          </div>
+          <div class="person">
+            <img
+              class="person__photo"
+              src="./images/people/3.jpg"
+              alt="Jonathan Ive"
+            >
+            <h3 class="person__name">
+              Jonathan Ive
+            </h3>
+            <p class="person__position">
+              Chief Design Officer
+            </p>
+            <p class="person__text">
+              When my lovely valley of steam rises and half-day sun is above an impermeable.
+            </p>
+          </div>
+        </div>
+      </section>
+      <section class="section">
+        <h2 class="heading section__heading" id="contact">
+          Contact Us
+          <span class="heading__crown"></span>
+        </h2>
+        <div class="contact">
+          <div class="contact__text">
+            <p>
+              <a href="mailto:Hello@miami.com" class="link">
+                Hello@miami.com
+              </a>
+              <br>
+              <a href="tel:+1-234-555-55-55" class="contact__number">
+                +1 234 555-55-55
+              </a>
+            </p>
+            <p>
+              <a
+                class="contact__address"
+                href="https://goo.gl/maps/sdiqcSyEs7jzAbD2A"
+                target="_blank"
+                rel="noopener noreferrer"
+              >
+              400 first ave. n.<br>suite 700<br>Minneapolis, MN 55401
+              </a>
+            </p>
+          </div>
+          <div>
+            <form
+              action="#"
+              method="POST"
+              class="form contact__form"
+            >
+              <label for="contact-name" class="form__item">
+                <input
+                  type="text"
+                  id="contact-name"
+                  name="name"
+                  placeholder="Name"
+                  class="input form__input"
+                >
+              </label>
+              <label for="contact-email" class="form__item">
+                <input
+                type="email"
+                id="contact-email"
+                name="email"
+                placeholder="Email"
+                class="input form__input"
+              >
+              </label>
+              <label for="message">
+                <textarea
+                  class="input input--textarea form__input"
+                  name="messege" id="message"
+                  placeholder="Messege"
+                >
+                </textarea>
```

Comment:
напиши все в одну строку, литер попропустит
```suggestion
                <textarea class="input input--textarea form__input" name="messege" id="message" placeholder="Messege"></textarea>
```